### DISCUSSION TITLE
Possible to change web console font and general appearance?

### DISCUSSION THREAD

**Discussion by Cyneric** [gravitational/teleport#33055] [Q&A] [answered]
Is it possible to adjust the background-color and the font of the web terminal?
Unfortunately I couldn't find something in the docs.

![image](https://github.com/gravitational/teleport/assets/6102978/01d1cb68-e0db-4bcb-9e53-45f542e80ba8)

**Comment by zmb3** [comment, marked as answer]
It's not possible to change the font.

The web UI does have a dark theme, which will change the color of the terminal background (any many other things)

The Teleport Connect client offers a similar experience for SSH and will allow you to customize the font.
